Add free trial docs and header CTA (#1832)
snowplow/documentation: docs/get-started/free-trial/index.md

---
title: "Snowplow free trial"
sidebar_position: 3
sidebar_label: "Free trial"
description: "Start a free 14-day trial of Snowplow CDI Cloud, and learn about the product and capacity limitations that apply during the trial."
keywords: ["free trial", "Snowplow trial", "CDI Cloud", "14-day trial", "trial limitations", "getting started"]
date: "2026-06-16"
---

> The Snowplow free trial is a 14-day, fully managed trial of [Snowplow CDI Cloud](/docs/get-started/index.md#cdi-cloud). Snowplow hosts and manages the pipeline infrastructure, so you can collect, validate, and load behavioral data without setting up any cloud infrastructure yourself.

Reviewer: ```suggestion
The Snowplow free trial is a 14-day trial of [Snowplow CDI Cloud](/docs/get-started/index.md#cdi-cloud). Snowplow hosts and manages the pipeline infrastructure, so you can collect, validate, and load behavioral data without setting up any cloud infrastructure yourself.
```
I think the repetition is redundant

Author: Applied — dropped "fully managed" to remove the repetition.

Author: Applied — dropped "fully managed" to remove the repetition.

Use the trial to evaluate Snowplow end to end: define your data, send events from your application, and load validated data into a warehouse or lake destination.

## Start the trial

Sign up for the free trial on the [Snowplow website](https://snowplow.io/get-started/snowplow-free-trial).

> After you sign up, your trial environment is provisioned automatically. Provisioning is not instant: trial access becomes available once provisioning completes, which takes around 20 minutes.

Reviewer: ```suggestion
After you sign up, your trial environment is provisioned automatically. You will receive access instructions within around 20 minutes.
```

Author: Applied your suggested wording.

Author: Applied your suggested wording.

> The trial runs for 14 days. If you need more time, extensions are available on request.

Reviewer: On request where? Should we make it more concrete?

Author: Made it concrete — extensions now link to contact sales to request one.

Author: Made it concrete — extensions now link to contact sales to request one.

## Product limitations

Some features are not configurable during the trial:

> * Collector configuration changes are not available.

Reviewer: I would not lead with this, as it’s a pretty low-level thing and people trying Snowplow for the first time will likely not know what this is about. Maybe just omit this altogether.

Author: Removed the Collector configuration bullet entirely.

Author: Removed the Collector configuration bullet entirely.

> * First-party collector and custom domains are not available.

Reviewer: I would phrase more around customer value rather than what it is technically

Author: Good idea - will update

Author: Reworded to frame it around the capability: "First-party tracking using your own domain is not available."

Author: Reworded around the capability: "First-party tracking using your own domain is not available."

> * Advanced or non-self-serviceable infrastructure is not available during the trial. [Contact sales](https://console.snowplowanalytics.com/contact-us) to discuss your requirements.

Reviewer: ```suggestion
* Some destinations are not available during the trial. [Contact sales](https://console.snowplowanalytics.com/contact-us) to discuss your requirements.
```

Reviewer: No need to call out general product limitations :)

Author: Applied.

Author: Dropped the generic framing — the section now lists only the specific, customer-relevant limitations.

Author: Applied.

Author: Dropped the generic framing — the section now lists only the specific, customer-relevant limitations.

## Capacity limits

The trial includes a limited set of resources that do not scale:

* Destinations are limited to:

>   * one [Micro](/docs/api-reference/snowplow-micro/index.md) instance

Reviewer: ```suggestion
  * one [Micro](/docs/testing/snowplow-micro/index.md) instance
```

Author: Applied — now points to /docs/testing/snowplow-micro/.

Author: Applied — now points to /docs/testing/snowplow-micro/.

  * one [warehouse or lake destination](/docs/destinations/warehouses-lakes/index.md)
  * one [event forwarding destination](/docs/destinations/forwarding-events/index.md)

> * Rate limiting is enforced at the collector.

Reviewer: At what rate?

Author: We don't publish a specific rate for the trial, so I generalized it to "Collector-level rate limits apply." Happy to add a figure if there's one we can share publicly.

Author: Added the concrete figure: "Collector-level rate limits apply, capped at around 20 events per second."
